.github/workflows/build.yml

```diff
@@ -39,3 +39,45 @@ jobs:
 
       - name: Upload Coverage Report
         uses: codecov/codecov-action@v1
+
+  provisioner-localpv:
+    needs: ['lint', 'unit-test']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set Tag
+        run: |
+          BRANCH="${GITHUB_REF##*/}"
+          CI_TAG=${BRANCH#v}-ci
+          if [ ${BRANCH} = "master" ]; then
```

Comment:
Can we replace this with `develop` as it will be the main branch on this repo.

Reply:
Done.